qa/L0_model_config/autofill_noplatform/ensemble/has_backend/expected

```diff
@@ -0,0 +1 @@
+Ensemble model 'has_backend' must have platform type 'ensemble' and empty backend type
```

Comment:
add a new line here.

Reply:
The "expected" files don't have a new line, not sure if `grep` will still be correct after adding it.

Reply:
Is this true for all.  I wonder why I didn't come to this issue before.